docs/experimental/experimental.rst

```diff
@@ -1,52 +1,27 @@
 Experimental
 ============
```

Comment:
Thanks for already taking care of parts of the docs :). The Spox Integration page looks unedited.